docs/source/content/examples/7-labeling.ipynb

```diff
@@ -80,100 +86,235 @@
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "text/plain": "''"
      },
-     "execution_count": 8,
+     "execution_count": 5,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
-    "measurements = session.list_measurements(SOURCE_ID)\n",
-    "measurements[:3]"
+    "result.plot()\n"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Adding labels\n",
+    "\n",
+    "A label in MVG relates to a specific measurement, which are identifiable by the source ID and the timestamp. Each measurement can only have one label. The label itself has three components\n",
+    "\n",
+    "1. A label identification string\n",
+    "2. A severity level number\n",
+    "3. And notes\n",
+    "\n",
+    ".. note:: The label identification string is unique for each label and is case and spelling sensitive, i.e. `\"failure\"` and `\"Failure\"` are not interpreted as the same label. The severity level is an integer where a larger number indicates a more severe problem. They do not need to be the same for different instances of the same label. The notes exist for the end user to add extra information to a certain label and is not used by MVG in any way."
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "For this example let us say that there was a bearing failure of the machine in the first few measurements, so we will add some labels to the afflicted measurements. To do so we use the `MVG.create_label()` method."
+    "For this example, let us say that there was a bearing failure of the machine in the first measurement of the source, so we will add a label to the afflicted measurements. To do so, we use the `MVG.create_label()` method."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 9,
+   "execution_count": 6,
    "metadata": {},
    "outputs": [],
    "source": [
-    "for measurement in measurements[:3]:\n",
-    "    session.create_label(\n",
-    "        sid=SOURCE_ID,\n",
-    "        timestamp=measurement[\"timestamp\"],\n",
-    "        label=\"bearing_failure\",\n",
-    "        severity=3,\n",
-    "        notes=\"Imaginary bearing failure\"\n",
-    "    )"
+    "first_measurement = 1570273260\n",
+    "session.create_label(\n",
+    "    sid=SOURCE_ID,\n",
+    "    timestamp=1570273260,\n",
+    "    label=\"bearing_failure\",\n",
+    "    severity=3,\n",
+    "    notes=\"Imaginary bearing failure\"\n",
+    ")"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "It might be a good idea to label more than one measurement per source.\n",
```

Comment:
@SergioMDCB Did you mean "per source" or "per mode" ? 
If I read the paragraph below in Label Propagation feature , it says one label per mode segment

Reply:
In this particular case I meant per source mainly because in the earlier code line we only added one label per source. Therefore, I was intended to imply that one label is not enough. I will rephrase the sentence to make it understandable.